optimize/components/userguide/process-dashboards.md

```diff
@@ -30,7 +30,8 @@ Note the available views below:
 
 9. KPI Results: You can see the KPI’s current and target values, plus a link to the report itself.
 
-10. Clicking on each process on the list opens a default dashboard dedicated to this process.
+10. Clicking on each process on the list opens a default dashboard dedicated to this process. For more details,
+    please [click here.](./instant-preview-dashboards.md)
```

Comment:
I don't think we typically say "click here" in our docs, rather using the link in a more normal structured sentence. Maybe @christinaausley can confirm or has a better idea

Reply:
I subtly incorporated the link in the existing sentence now